project/royyao1997/README.md

```diff
@@ -0,0 +1,72 @@
+============================================================================
+Strassen Algorithm
+============================================================================
+
+
+
+Basic Information
+=================
+
+The Strassen algorithm, is an algorithm for matrix multiplication. It is faster than the standard matrix multiplication algorithm for large matrices, with a better asymptotic complexity.
```

Comment:
While the topic perfectly align with numerical calculation, it will be difficult for you to come up with a Python API other than `multiply()` :)  The topic heavily focuses on the numerical algorithm instead of software engineering.  You need to think hard to make a good presentation on the software part.

Designing an algorithm and a software structure is complex, but implementing an algorithm is simpler, because the latter involves less trial-and-error.